content/source/docs/enterprise/api/oauth-clients.html.md

```diff
@@ -111,6 +111,100 @@ curl \
 }
 ```
 
+## Update an OAuth Client
+
+`PATCH /oauth-clients/:id`
+
+Parameter            | Description
+---------------------|------------
+`:id`                | The ID of the OAuth Client to update.
+
+Result  | Status and response
+--------|------------------------
+Success | HTTP 200 and a JSON API document (`type: "oauth-clients"`).
+Failure | HTTP 4XX and a JSON API list of error messages.
```

Comment:
To be consistent, we should list all the specific 400s (or at least all the common ones).

Reply:
The Status/Response section has been replaced on all endpoints in this doc.